java/hsfs/src/main/java/com/logicalclocks/hsfs/FeatureStore.java

```diff
@@ -0,0 +1,118 @@
+package com.logicalclocks.hsfs;
+
+import com.logicalclocks.hsfs.constructor.Query;
+import com.logicalclocks.hsfs.engine.FeatureGroupEngine;
+import com.logicalclocks.hsfs.engine.FeatureViewEngine;
+import lombok.NonNull;
+
+import java.io.IOException;
+
+public class FeatureStore extends FeatureStoreBase<Query> {
+
+  private FeatureGroupEngine featureGroupEngine;
+  private FeatureViewEngine featureViewEngine;
+
+  public FeatureStore() {
```

Comment:
General question: what methods should be included in the java client?

Reply:
Only thing in java client can be done is fetch FG, FV metadata and get feature vectors. I don't see any other use cases